doc/ADMIN.md

```diff
@@ -2,4 +2,5 @@
 
 To ease the install process and first access, Tiki saves your uploaded files (office documents, images, pdf, etc. attached to wiki pages, forum posts, tracker items, file galleries...) by default in its database. This works perfectly in most cases but it is not the recommended setup if you need to save many thousands of files or more.
 
-In that case, consider switching from "Store to database" to "Store to directory", which you will find in the Configuration Wizard. Please use this preset path directory: `/home/yunohost.app/$app`. You will be able to migrate your currently uploaded files from one to the other.
+In that case, consider switching from "Store to database" to "Store to directory", which you will find in the Configuration Wizard. Please use the preset path directory: `/home/yunohost.app/tiki<…>`. You will be able to migrate your currently uploaded files from one to the other.
```

Comment:
```suggestion
In that case, consider switching from "Store to database" to "Store to directory", which you will find in the Configuration Wizard. Please use the preset path directory: `__DATA_DIR__`. You will be able to migrate your currently uploaded files from one to the other.
```

Reply:
with tha app v2, we are now able to get the proper path with `__DATA_DIR__ set in `ADMIN.md`